features/openhab-addons/src/main/resources/footer.xml

```diff
@@ -1,2 +1,11 @@
+    <feature name="openhab-misc-ruleengine" description="Rule Engine (Experimental)" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
Hm. I think we should remove this feature.
Usually an extension only depends on "openhab-core-automation".

The java sound extensions might also pull in "openhab-core-automation-module-media" if they really want to, but they do not depend on those.

@kaikreuzer @maggu2810 @wborn Can we add "openhab-core-automation-module-script" and "openhab-core-automation-module-script-rulesupport" to the core automation feature? (Is there a reason for the split?)

Reply:
Is there a reason why products should be forced to use the script bundles if they want to use automation without script support in their runtime?
So, yes, there is a reason. The core automation feature should not contain optional support like the one for scripts.

Reply:
I see. But script support (this is not script engines, but script file loading I guess?) is nothing that ever will be required as dependency by any of the extensions here. So it's safe to be removed as feature here in the addons repo.